clap-v3-utils/src/keypair.rs

```diff
@@ -807,7 +807,9 @@ pub fn signer_from_path_with_config(
             }
         }
         SignerSourceKind::Pubkey(pubkey) => {
-            let presigner = pubkeys_sigs_of(matches, SIGNER_ARG.name)
+            let presigner = try_pubkeys_sigs_of(matches, SIGNER_ARG.name)
+                .ok()
```

Comment:
```suggestion
            let presigner = try_pubkeys_sigs_of(matches, SIGNER_ARG.name)?
```

right? otherwise we hide the error, which may confuse the user

Reply:
Yeah, that was what I was mentioning in the PR description. The issue is that `SIGNER_ARG.name` is not defined in some cli tools (well... I think just the token-cli). If we immediately return an error here, then there is no way for downstream cli tools to execute the presigner logic below whereas, if we convert to `Option`, then the behavior stays the same and the presigner logic can be executed.

Reply:
Thinking about it now though... it seems like propagating the error out is the right way forward with the clap-v3... I will just add the `SIGNER_ARG.name` in the token-cli itself.

Reply:
Your comment suggests you were planning to take Trent's suggestion. Is that right? I don't see that change and wondering if I'm missing a push

Reply:
Oh sorry! I must have overwritten the commit when I rebased. I just pushed the commit now.